docs/basics.md

```diff
@@ -86,6 +86,7 @@ Following is the list of existing modifier rules:
 
 - `AddPrefix: /products`: Add path prefix to the existing request path prior to forwarding the request to the backend.
 - `ReplacePath: /serverless-path`: Replaces the path and adds the old path to the `X-Replaced-Path` header. Useful for mapping to AWS Lambda or Google Cloud Functions.
+- `ReplacePathRegex: ^/prefix $> /api`: Replaces the path with a regular expression and adds the old path to the `X-Replaced-Path` header, the operator `$>` means `Replace to`.
```

Comment:
I don't really like the ` $> ` syntax.
Maybe we can use the `sed` syntax  `{sep}regex{sep}replacement{sep}`.
ex.
```
ReplacePathRegex:%/whoami/(.*)%/who-am-i/$1%
```

@containous/traefik  WDYT ?

Reply:
I tend to agree with your approach @Juliens. It looks more familiar and therefore easier to use. Also, I am not sure about possible ambiguity in regards to whitespaces in the `^/prefix $> /api` approach.

Reply:
How about the syntax likes [nginx http rewrite module](http://nginx.org/en/docs/http/ngx_http_rewrite_module.html)?
`ReplacePathRegex: (?i)^/whoami/(.*) /who-am-i/$1`